lib/streamlit/runtime/fragment.py

```diff
@@ -174,12 +175,10 @@ def wrapped_fragment():
         ctx.fragment_storage.set(fragment_id, wrapped_fragment)
 
         if run_every:
-            # TODO(vdonato): Fix me when implementing auto rerun.
-            # msg = ForwardMsg()
-            # msg.auto_rerun.interval = run_every
-            # msg.auto_rerun.fragment_id = fragment_id
-            # ctx.enqueue(msg)
-            pass
+            msg = ForwardMsg()
+            msg.auto_rerun.interval = run_every
```

Comment:
nit: I think it was discussed that  `run_every` should eventually handle the same types as the caching `ttl` parameter (string, timedelta). But I don't think it necessarily needs to be in the first version. Maybe something to double-check with Joshua.